Tests/FluentAssertions.Equivalency.Specs/MemberConversionSpecs.cs

```diff
@@ -51,6 +51,35 @@ public void When_an_int_is_compared_equivalent_to_a_string_representing_the_numb
         act.Should().NotThrow();
     }
 
+    [Fact]
+    public void Numbers_can_be_converted_to_enums()
+    {
+        // Arrange
+        var expectation = new { Property = EnumFour.Three };
+        var subject = new { Property = 3 };
+
+        // Act
+        Action act = () => subject.Should().BeEquivalentTo(expectation, options => options.WithAutoConversion());
+
+        // Assert
+        act.Should().NotThrow();
```

Comment:
You could omit the `Action` and call it directly in that case

like:
```csharp
// Act / Assert
subject.Should().BeEquivalentTo(expectation, options => options.WithAutoConversion());
```

Reply:
Yes, that's true. We don't explicitly verify that an API does _not_ throw.